community/examples/htcondor-pool.yaml

```diff
@@ -0,0 +1,136 @@
+  deployment_name: htcondor-001
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: htcondor
+  modules:
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+    settings:
+      network_name: htcondor-pool
+      subnetwork_name: htcondor-pool-usc1
+    outputs:
+    - network_name
+
+  - source: community/modules/scripts/htcondor-install
+    kind: terraform
+    id: htcondor_install
+
+  - source: community/modules/project/service-enablement
+    kind: terraform
+    id: htcondor_services
+    use:
+    - htcondor_install
+
+  - source: community/modules/scheduler/htcondor-configure
+    kind: terraform
+    id: htcondor_configure
+
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_configure_central_manager
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+      - $(htcondor_configure.central_manager_runner)
+
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_configure_access_point
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+      - $(htcondor_install.install_autoscaler_deps_runner)
+      - $(htcondor_install.install_autoscaler_runner)
+      - $(htcondor_configure.access_point_runner)
+      - $(htcondor_execute_point.configure_autoscaler_runner)
+
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_configure_execute_point
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+      - $(htcondor_configure.execute_point_runner)
+
+  - source: modules/compute/vm-instance
+    kind: terraform
+    id: htcondor_cm
+    use:
+    - network1
+    - htcondor_configure_central_manager
+    settings:
+      enable_oslogin: DISABLE
+      name_prefix: central-manager
+      machine_type: c2-standard-4
+      disable_public_ips: true
+      service_account:
+        email: $(htcondor_configure.central_manager_service_account)
+        scopes:
+        - cloud-platform
+    outputs:
+    - internal_ip
+
+  - source: community/modules/compute/htcondor-execute-point
+    kind: terraform
+    id: htcondor_execute_point
+    use:
+    - network1
+    - htcondor_configure_execute_point
+    settings:
+      metadata:
+        central-manager: ((module.htcondor_cm.internal_ip[0]))
+        enable-oslogin: "FALSE"
+      service_account:
+        email: $(htcondor_configure.execute_point_service_account)
+        scopes:
+        - cloud-platform
+
+  - source: modules/compute/vm-instance
```

Comment:
No action: Part of me wants to see a htcondor access and htcondor cm modules. It would definitely be possible to make variable passing easier, cut down on some verbosity, and maybe even take the startup scripts inside of the module. With that said I think what you have here gets it down with much less overhead, so no objection.

Reply:
Upon original design we had not yet used GitHub URL based approach to use Toolkit modules. There is a need for enforcing the `central-manager` metadata existence and correctness. Maybe that could be addressed with `use`. I'll take a look / create a task.

Reply:
Ack